plugins/modules/rds_instance.py

```diff
@@ -1394,6 +1403,11 @@ def main():
         supports_check_mode=True
     )
 
+    if module.params["ca_certificate_identifier"]:
+        module.require_botocore_at_least(
+            "1.29.44", reason="to use 'ca_certificate_identifier' while creating/updating rds instance"
```

Comment:
Can you please add a note in the documentation about botocore requirements for this parameter?

Reply:
added